### ISSUE TITLE
SystemJS is not (ye) supported on Nashorm

### ISSUE THREAD

**Issue by manekinekko** [manekinekko/angular2-springboot-engine#1]
see https://github.com/systemjs/systemjs/issues/1113

**Comment by TheLarkInn** [comment]
@manekinekko have you attempted a webpack build instead?

**Comment by manekinekko** [comment]
Nope. That was my next step. But please feel free to try this out

**Comment by TheLarkInn** [comment]
I'm going to show a co-worker this and see if he can set it up and then I'll work the webpack config, and I'll get back to.
